src/platform/echarts/scatter/scatter.component.ts

```diff
@@ -0,0 +1,156 @@
+import {
+  Component,
+  Input,
+  OnChanges,
+  OnInit,
+  OnDestroy,
+  ChangeDetectionStrategy,
+} from '@angular/core';
+
+import 'echarts/lib/chart/scatter';
+import 'echarts/lib/component/markPoint';
+import 'echarts/lib/component/markLine';
+import 'echarts/lib/component/markArea';
```

Comment:
I dont wanna import `markPoint`, `markLine` nor `markArea` with this component.. we will either have it in the demos on what to import in the product, or create a subcomponent for each one that does it.

Reply:
@emoralesb05 OK so we will leave it up to the developer to add these in at the application component level as needed, correct? Make sense as you only need to add them once it seems work across the entire app once added. Same with `import 'echarts/lib/component/tooltip';` I'm guessing then?